showcase/app/styles/showcase-pages/application-state.scss

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// APPLICATION-STATE
+
+body.application-state {
+  .shw-avatar {
+    height: 125px;
+    width: 125px;
+    border-radius: 100%;
+    overflow: hidden;
```

Comment:
`overflow: hidden` shouldn't be needed as images don't have content which can overflow. (It _is_ possible for alt text to overflow but only if overflow is purposely set to `visible`.)